Provide indication of stale platforms and ability to update them

jsoref opened the pull request:
The refresh icon is derived from this public domain icon:
 http://publicdomainvectors.org/en/free-clipart/Eco-water-recycling-vector-icon/6856.html
 -- it's actually only inspired by, this art was drawn by me using InkScape

Closes #7

jsoref committed:
Provide indication of stale platforms and ability to update them

The refresh icon is derived from this public domain icon:
 http://publicdomainvectors.org/en/free-clipart/Eco-water-recycling-vector-icon/6856.html
 -- it's actually only inspired by, this art was drawn by me using InkScape

rcruz commented on lib/ui/views/updatesList.js:
```diff
@@ -0,0 +1,85 @@
+
+function updatePlatform(config, platform, display) {
+    var newObj = true;
+}
```
Unused and mysterious function

rcruz commented:
Functions well. Just need to match appearance with the rest of the page: font color #565e66, border of button #cccccc, button hover background color #cccccc

rcruz commented:
@bryanhiggins @kwallis When upgrading a platform is possible, do we want to encourage the user to update? I'm thinking a highlight color on the upgrade notification will help with this. Thoughts?

kwallis commented:
How risky is an upgrade? How often would it require code changes? 

jsoref commented:
@rcruz I think a highlight color is probably a good idea.

@kwallis within the **WebWorks** 1..2 cycle, we've occasionally messed up such that upgrading broke things (specifically Cordova's `exec` method is supposed to be **async**, but we had written a bunch of plugins that expected it to be **sync**, and when we fixed our `exec` to be **async** per spec, it broke all of our buggy plugins.) -- this was fixed before we shipped (well, for a weak definition of _fixed_).

OTOH, we also fixed a bunch of things for the 2.0 cycle such that in general an upgrade was a safe (to the best of our knowledge) action.

bryanhiggins commented:
@kwallis The upgrade only affects the cordova-blackberry library, not the plugins, so it should never (or very rarely) require code changes. There is a story in the backlog to show plugins which have an update available.

kwallis commented:
OK, let's highlight.
